packages/fx-core/resource/package.nls.json

```diff
@@ -187,6 +187,12 @@
   "plugins.spfx.questions.versionCheck.title": "Checking Node.js and NPM versions...",
   "plugins.spfx.questions.webpartName.error.duplicate": "The folder %s already exists. Choose a different name for your component.",
   "plugins.spfx.questions.webpartName.error.notMatch": "%s doesn't match pattern: %s",
+  "plugins.spfx.questions.packageSelect.title": "SharePoint Framework",
+  "plugins.spfx.questions.packageSelect.placeholder": "Select an option for scaffolding",
+  "plugins.spfx.questions.packageSelect.useGlobalPackage.withVersion.label": "Use globally installed SPFx (%s)",
```

Comment:
Add 'v' prefix? Need PM confirm here.

Reply:
updated. thought it would be consistent with the description message we put.

Reply:
updated the code again. and will add "v" in code when generating string. Please see the attached screenshot in PR description.